ssl/t1_lib.c

```diff
@@ -1498,7 +1498,7 @@ int tls12_check_peer_sigalg(SSL *s, uint16_t sig, EVP_PKEY *pkey)
 
         /* For TLS 1.3 or Suite B check curve matches signature algorithm */
         if (SSL_IS_TLS13(s) || tls1_suiteb(s)) {
-            int curve = evp_pkey_get_EC_KEY_curve_nid(pkey);
```

Comment:
And here it is enclosed in ifndef OPENSSL_NO_EC as well. I am sorry for taking you to this path. Maybe this should have been deferred to a separate PR.

Reply:
Yes, I know...  there's a call in `ssl/statem/statem_lib.c` as well, and quite frankly, I can't see the consequences clearly enough to dare fix them all unless someone can tell me for sure that I have nothing to worry about (@mattcaswell is my guru in this, but if you are confident that you see clearly enough, I'll trust you).

Reply:
I am afraid there might be some cascading effect. So probably better to defer this. I've opened #13616 to cover this in general.

Reply:
Cool.  Then I'll resolves these last comments unchanged.

Reply:
> I am afraid there might be some cascading effect.

I share the feeling.

Thank you, I'll resolve these remaining concerns unchanged, then.